src/cli/rustup_mode.rs

```diff
@@ -8,12 +8,13 @@ use std::str::FromStr;
 use anyhow::{anyhow, bail, Error, Result};
 use clap::{App, AppSettings, Arg, ArgGroup, ArgMatches, Shell, SubCommand};
 
+use super::common;
 use super::help::*;
 use super::self_update;
 use super::term2;
 use super::term2::Terminal;
 use super::topical_doc;
-use super::{common, self_update::get_available_rustup_version};
+use crate::cli::self_update::{check_rustup_update, SELF_UPDATE_MODES};
```

Comment:
Why the change from super::... to crate::cli::... ?

I'm not against it, just curious?

```
use super::{common, self_update::{check_rustup_update, SELF_UPDATE_MODES}}
```

would have worked fine

Reply:
Oops! My IDE imported it for me. Fixed.